<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaxViewer</class>
 <widget class="QWidget" name="FaxViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Просмотр изображений</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="saveButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Сохранить изображение</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="printButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Печать изображения</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="plusButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Увеличить изображение</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/meteo/icons/tools/faxview/zoom_in.png</normaloff>:/meteo/icons/tools/faxview/zoom_in.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="minusButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Уменьшить изображение</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/meteo/icons/tools/faxview/zoom_out.png</normaloff>:/meteo/icons/tools/faxview/zoom_out.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="originalSizeButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Показать в оригинальном размере</string>
       </property>
       <property name="text">
        <string>Or</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/meteo/icons/tools/faxview/zoom_original.png</normaloff>:/meteo/icons/tools/faxview/zoom_original.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="windowSizeButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Подогнать по размеру окна</string>
       </property>
       <property name="text">
        <string>Sq</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/meteo/icons/tools/faxview/zoom_best.png</normaloff>:/meteo/icons/tools/faxview/zoom_best.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clockwiseButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Повернуть на 90&lt;span style=&quot; vertical-align:super;&quot;&gt;0&lt;/span&gt; по часовой стрелке&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Cl</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/meteo/icons/tools/faxview/rotate_clockwise.png</normaloff>:/meteo/icons/tools/faxview/rotate_clockwise.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="anticlockwiseButton">
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Повернуть на 90&lt;span style=&quot; vertical-align:super;&quot;&gt;0&lt;/span&gt; против часовой стрелки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Ac</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/meteo/icons/tools/faxview/rotate_anticlockwise.png</normaloff>:/meteo/icons/tools/faxview/rotate_anticlockwise.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGraphicsView" name="imageGraphicsView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
